css/style.css

```diff
@@ -507,33 +505,149 @@ address {
     margin-top: 0em;
 }
 
+.floe .idrc-logo-container svg,
+.floe .hewlett-logo-container svg {
+    padding: 0 0.5em 0 0.5em;
+    width: 100%;
+    height: auto;
+}
+
+/* IE-only hacks for better SVG sizing */
+
+/* Large, two column */
+@media all and (min-width: 64em) and (-ms-high-contrast:none)
+ {
+     .floe .idrc-logo-container svg,
+     .floe .hewlett-logo-container svg {
+        height: 50px;
+     }
+ }
+
+/* Medium, one column */
+
+ @media all and (min-width: 40.063em) and (max-width: 64em) and (-ms-high-contrast:none)
+  {
+      .floe .idrc-logo-container svg,
+      .floe .hewlett-logo-container svg {
+         height: 150px;
+      }
+  }
+
+/* Small, one column */
+
+  @media all and (max-width: 40em) and (-ms-high-contrast:none)
```

Comment:
Same comment as above about using `active` and `none`. Also it may be worthwhile to comment if this is specific for IE and/or Edge.